### PR TITLE
PBT - Enhancements

### PR DESCRIPTION
**Pull request by maksymhubenko**

### CODE REVIEW

**Comment by maksymhubenko** [comment]
@dvasyab @yugisu QuickTheories library should be removed in the nearest future (unless, of course, used).

File: src/test/java/url/shortener/server/service/impl/UrlServiceImplTestPropertyBased.java [hunk 1]
```diff
@@ -1,153 +1,188 @@
...
+        methodName,
+        successCounter,
+        REPEATS,
+        (float) successCounter / REPEATS * 100
+    );
   }
 
-  @AfterEach
-  void removeDBFolder() {
+//  @BeforeEach
```
Comment (IINamelessII) [inline src/test/java/url/shortener/server/service/impl/UrlServiceImplTestPropertyBased.java]: do not leave commented code wo a comment why it's needed
or just remove it pls

**Review by IINamelessII — CHANGES_REQUESTED** [review]

File: src/test/java/url/shortener/server/service/impl/UrlServiceImplTestPropertyBased.java (hunk 1 above)
Comment (maksymhubenko) [reply src/test/java/url/shortener/server/service/impl/UrlServiceImplTestPropertyBased.java]: Got you.

**Review by IINamelessII — APPROVED** [review]

**Comment by dvasyab** [comment]
As for me, that is all necessary changes that we had to done.

**Review by dvasyab — APPROVED** [review]

**Review by IINamelessII — APPROVED** [review]
Yes.

**Review by yugisu — APPROVED** [review]

**Pull request merged**